components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/AbstractAPIManager.java

```diff
@@ -539,4 +540,9 @@ protected void addWorkflowEntries(Workflow workflow) throws APIManagementExcepti
         }
 
     }
+    
+    @Override
+    public Label getLabelByName(String labelName) throws APIManagementException {
+        return labelDAO.getLabelByName(labelName);
+    }
```

Comment:
Don't we need to catch APIMgtDAOException exception which is thrown from getLabelByName method?